build.gradle

```diff
@@ -60,3 +60,36 @@ allprojects {
 task clean(type: Delete) {
     delete rootProject.buildDir
 }
+
+apply plugin: 'maven-publish'
+
+def getArtificatId = { ->
+    return "rxandroidble"
+}
+
+def getArtificatVersion = { ->
+    return "1.1"
```

Comment:
can we sync this with the original library versioning? This way we can keep track